src/Components/IndexScene/IndexScene.tsx

```diff
@@ -220,7 +249,6 @@ function getVariableSet(initialDatasourceUid: string, initialFilters?: AdHocVari
     applyMode: 'manual',
     layout: 'vertical',
     getTagKeysProvider: () => Promise.resolve({ replace: true, values: [] }),
-    getTagValuesProvider: () => Promise.resolve({ replace: true, values: [] }),
```

Comment:
OHHHHHH. I'm am very certain that it would have been way easier to spot if this wasn't called "tags".